src/main/java/com/faforever/client/replay/ReplayDetailController.java

```diff
@@ -321,6 +315,22 @@ private void populateTeamsContainer() {
     }));
   }
 
+  private void configureRatingControls() {
+    if (!replay.getValidity().equals(Validity.VALID)) {
+      showRatingChangeButton.setVisible(false);
+      notRatedReasonLabel.setVisible(true);
+      notRatedReasonLabel.setText(i18n.get("game.reasonNotValid", i18n.get(i18n.get("game.reasonNotValid.keyFormat", replay.getValidity().ordinal()))));
```

Comment:
I am pretty sure game.reasonNotValid.keyFormat should not be in I18N, imagine somebody thinks he most translate it then this breaks for the other languages.

Reply:
Hmm yeah good point will just change it back to string formatting unless you think there is a better way